nucleus/__init__.py

```diff
@@ -128,7 +128,7 @@ def __init__(self, api_key: str, use_notebook: bool = False):
         self.api_key = api_key
         self.tqdm_bar = tqdm.tqdm
         if use_notebook:
-            self.tqdm_bar = tqdm.notebook.tqdm
+            self.tqdm_bar = tqdm_notebook.tqdm
```

Comment:
Was running into issues calling the api from within a notebook, this resolved it for me